### PR TITLE
Configure holochain at each restart

### PR DESCRIPTION
**Pull request by peeech**
tries to address #635

### CODE REVIEW

**Review by Br1ght0ne — APPROVED** [review]
LGTM.

File: modules/services/self-hosted-happs.nix [hunk 1]
```diff
@@ -31,6 +31,7 @@ in
       after = [ "network.target" "holochain.service" ];
       bindsTo = [ "holochain.service" ];
       wantedBy = [ "multi-user.target" ];
+      reloadPropagatedFrom = [ "holochain.service" ];
```
Comment (Br1ght0ne) [inline modules/services/self-hosted-happs.nix:34]: Are there any docs on this option? Can't seem to find anything.

Comment (peeech) [reply modules/services/self-hosted-happs.nix:34]: `man systemd.unit`, but [nixpkgs systemd wrapper](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/system/boot/systemd.nix) does not support this option. 

Still this won't work the way I'd like it to. I am gonna describe an issue in #635

Comment (Br1ght0ne) [reply modules/services/self-hosted-happs.nix:34]: Can't we just use `serviceConfig` then?

```
serviceConfig.ReloadPropagatedFrom = "holochain.service";
```

**Comment by peeech** [comment]
This is a wrong direction. Closing.

**Pull request closed without merging**